README.md

```diff
@@ -1 +1,11 @@
-# HydraDX-audits
\ No newline at end of file
+# HydraDX-audits-reports
+This repository contains the reports produced from the audits of the HydraDX Protocol.
+
+## Runtime Audit
+Conducted by [Runtime Verification](https://runtimeverification.com/), published in September 2022.  
+Read the full report [here](220907-Runtime-Verification-Audit.pdf).
+
+## Economic and Mathematical Audit
+Conducted by [BlockScience][https://block.science/], published in March 2022.  
+Addendum by the HydraDX team elaborating on some changes which were made after the audit was finished (pp 41 et seq), published in November 2022.  
+Read the full report [here](220322-BSCI-Audit+addendum-by-HydraDX.pdf).
```

Comment:
This is minor but can we rename this report, can be just "Block Science Report on Omnipool" or something. I put zero thought into the name of the pdf currently

Reply:
yep lgtm